lib/ui.js

```diff
@@ -3,23 +3,32 @@ const path = require('path')
 const serve = require('serve-static')
 
 module.exports.serveSwaggerUI = function serveSwaggerUI (documentUrl) {
-  return [serve(path.resolve(require.resolve('swagger-ui-dist'), '..'), { index: false }), function renderSwaggerHtml (req, res) {
-    res.type('html').send(renderHtmlPage('Swagger UI', `
+  return [serve(path.resolve(require.resolve('swagger-ui-dist'), '..'), { index: false }),
+    function returnUiInit (req, res, next) {
```

Comment:
Maybe consider declaring the middleware function outside so it's cleaner.

Reply:
I think it makes sense but given that the other function is also inline, it might be better to refactor it as well, and I prefer to keep it as is to be able to test it faster, because I have the feeling I will have to get back to this later